app/src/components/RssiHeightGraph.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+import * as React from 'react';
+import {useState} from 'react';
+import Button from '@material-ui/core/Button';
+import {
+  XYPlot,
+  XAxis,
+  YAxis,
+  HorizontalGridLines,
+  VerticalGridLines,
+  MarkSeries,
+} from 'react-vis';
+import '../../node_modules/react-vis/dist/style.css';
+
+import type {LayerDict} from './MapScreen';
```

Comment:
should this be '../screens/MapScreen'?  When I checked out your branch I had an error here

Reply:
Yup! Good catch.
I moved it after it's initial placement, and VSCode asked if I wanted to autoupdate. Guess it missed this one.
I supposed I didnt reopen the file, and never got the flow errors...